cmd/utils/nodecmd/db_migration.go

```diff
@@ -0,0 +1,157 @@
+// Modifications Copyright 2020 The klaytn Authors
+// Copyright 2015 The go-ethereum Authors
+// This file is part of go-ethereum.
+//
+// go-ethereum is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// go-ethereum is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with go-ethereum. If not, see <http://www.gnu.org/licenses/>.
+
+package nodecmd
+
+import (
+	"encoding/json"
+
+	"github.com/klaytn/klaytn/cmd/utils"
+	"github.com/klaytn/klaytn/storage/database"
+	"github.com/pkg/errors"
+
+	"gopkg.in/urfave/cli.v1"
+)
+
+var (
+	dbFlags = []cli.Flag{
+		// src DB
+		utils.DbTypeFlag,
+		utils.SingleDBFlag,
+		utils.NumStateTrieShardsFlag,
+		utils.DynamoDBTableNameFlag,
+		utils.DynamoDBRegionFlag,
+		utils.DynamoDBIsProvisionedFlag,
+		utils.DynamoDBReadCapacityFlag,
+		utils.DynamoDBWriteCapacityFlag,
+		utils.LevelDBCompressionTypeFlag,
+		utils.DataDirFlag,
+	}
+	dbMigrationFlags = append(dbFlags, DBMigrationFlags...)
+
+	MigrationCommand = cli.Command{
+		Name:     "db-migration",
+		Usage:    "db migration",
+		Flags:    []cli.Flag{},
+		Category: "DB MIGRATION COMMANDS",
+		Description: `
+The migration commands migrates a DB to a different kind of DB.
```

Comment:
Is this command not used for the same kind of DB?

Reply:
It can be used for same kind of DB. I've changed the description. PTAL :)